### PR TITLE
kraken: manage line_section and routes

### PR DESCRIPTION
**Pull request by kadhikari**
https://jira.kisio.org/browse/NAVITIAII-2781

Needs https://github.com/CanalTP/navitia/pull/2863 to test disruption charging from a Postgres database with a backup restored and containing somes disruptions on stop_area, line_section, deviation...

### CODE REVIEW

**Comment by kadhikari** [comment]
Jenkins, test this please

**Review by kinnou02 — APPROVED** [review]

File: source/kraken/fill_disruption_from_database.h [hunk 1]
```diff
@@ -73,9 +73,8 @@ struct DisruptionDatabaseReader {
 
     std::set<std::string> message_ids;
     std::set<std::tuple<std::string, std::string, std::string>> properties;
-    std::set<std::string> pt_object_ids;
```
Comment (benoit-bst) [inline source/kraken/fill_disruption_from_database.h:76]: I guess it was not used

File: source/kraken/fill_disruption_from_database.h [hunk 2]
```diff
@@ -102,9 +101,9 @@ struct DisruptionDatabaseReader {
             message = nullptr;
             channel = nullptr;
             pt_object = nullptr;
```
Comment (benoit-bst) [inline source/kraken/fill_disruption_from_database.h:103]: minor: don't need to `nullptr` [property](https://github.com/CanalTP/navitia/pull/2848/files#diff-6e63c97d3884a53ade05aea2c2d061c0R70) too ?

Comment (kadhikari) [reply source/kraken/fill_disruption_from_database.h:103]: This part of code is not modified in this PR.

File: source/kraken/fill_disruption_from_database.h (hunk 1 above)
Comment (kadhikari) [reply source/kraken/fill_disruption_from_database.h:76]: This variable was used before, but not anymore.

**Review by nonifier — APPROVED** [review]

**Review by benoit-bst — APPROVED** [review]

**Pull request merged**